client/lib/plans/features-list.js

```diff
@@ -76,10 +85,14 @@ export const FEATURES_LIST = {
 			i18n.translate( '{{a}}All Premium features{{/a}}', {
 				components: {
 					a: (
-						<a
+						<ExternalLinkWithTracking
 							href="https://jetpack.com/features/comparison"
 							target="_blank"
-							rel="noopener noreferrer"
+							tracksEventName="calypso_plan_link_click"
+							tracksEventProps={ {
+								link_location: 'plan_features_list_item',
+								plan_slug: constants.PLAN_JETPACK_PREMIUM,
```

Comment:
If you ask me, these `plan_slug` are still confusing in this context. Should we somehow map this to the `FEATURE_ALL_PREMIUM_FEATURES_JETPACK` feature instead of to a plan?